src/home/RegisteredView.tsx

```diff
@@ -70,11 +73,14 @@ export function RegisteredView({ client, isPasswordlessUser }: Props) {
         setError(undefined);
         setLoading(true);
 
-        const [roomAlias] = await createRoom(client, roomName, ptt);
+        const [roomAlias, roomId] = await createRoom(client, roomName, ptt);
 
-        if (roomAlias) {
-          history.push(`/${roomAlias.substring(1).split(":")[0]}`);
-        }
+        setLocalStorageItem(
+          getRoomSharedKeyLocalStorageKey(roomId),
+          randomString(32)
+        );
```

Comment:
If we're storing one item in localstorage per room, I think we need to handle running out of space unfortunately: I'm not sure how though, I guess we would have to start forgetting old rooms? Also, what happens in firefox private mode?

Reply:
From what I found online, there is a 5 MB - 10 MB limit though I didn't find anything about an item count limit?

Reply:
> Also, what happens in firefox private mode?

The non-nightly versions of FF still don't have insertable streams, so it doesn't matter much atm...

Reply:
But it's still going to fail after you joined a certain number of e2ee rooms, whether the limit is in in count or bytes?

Do we at least know what the failure mode is without localstorage?